source/schemas/types.txt

```diff
@@ -415,6 +428,39 @@ BSON type <https://bsonspec.org/spec.html>`__. Always uses subtype 0.
 Realm Database Types
 --------------------
 
+Realm Database types are specific to Atlas Device SDK. 
+
+For more information on SDK-specific types and how to define them,
+refer to the SDK documentation:
+
+- :ref:`cpp-supported-property-types`
+- :ref:`flutter-data-types`
+- :ref:`java-data-types`
+- :ref:`kotlin-supported-types`
+- :ref:`dotnet-field-types`
+- :ref:`node-data-types`
+- :ref:`react-nativedata-types-field-types`
+- :ref:`<ios-supported-property-types>`
+
+.. _schema-type-counter:
+
+Counter
```

Comment:
new Counter data type section (wasn't documented)

Reply:
Thank you for adding this :)